### ISSUE TITLE
Guidance on usage: Formats and architecture

### ISSUE THREAD

**Issue by SreehariSankar** [msobrevillac/pretrained-amr-to-text#1]
Hello Marco,
Could you please provide some guidance on how to use this. Specifically, how the inputs are to be structured and the architecture of the actual converter? I presume you are using some kind of LSTM? If so, do you provide the encoder/decoder?

Sincerely,
Hari

P.S:
Please let me know if you want a collaborator on this. I have been trying to get a working AMR-->Text converter for a long time now.

**Comment by msobrevillac** [comment]
Hi @SreehariSankar, I am still developing some parts of the code. Here is an example of the training
python Train.py -train-src processed_data/amr/pt/penman/train/sentence.txt -train-tgt processed_data/amr/pt/penman/train/amr.txt \
    -dev-src processed_data/amr/pt/penman/dev/sentence.txt -dev-tgt processed_data/amr/pt/penman/dev/amr.txt \
    -test-src processed_data/amr/pt/penman/test/sentence.txt \
    -batch-size 16 -max-length 150 -lr 5e-4 \
    -epochs 3 -gpu -print-every 300 -model pierreguillou/gpt2-small-portuguese \
    -save-dir output/ -seed 42 -beam-size 15 -representation amr -fixed-embed

could you send me an email to make a contact or talking about your work?
